api-doc.md

```diff
@@ -298,3 +298,42 @@
             </url>
         </urlset>
 ```
+
+### 4. Send New Leave Request mail
+
+```
+- Method : POST
+- Endpoint : /api/leave/new
+- Description : API for sending mail to HR for New leave request
+- Request:
+    - Headers : none
+    - Body :
+        {
+            "name": "mansi dhameliya",
+            "date": "3 jan 2023",
+            "status": "request",
```

Comment:
Any need of status?

Reply:
for display status : request,approve,reject,etc